powermon/protocols/pi30.py

```diff
@@ -581,23 +582,23 @@
         "name": "QPI",
         "description": "Protocol ID inquiry",
         "help": " -- queries the device protocol ID. e.g. PI30 for HS series",
-        "response_type": ResponseType.INDEXED,
-        "response": [[0, "Protocol ID", "bytes.decode", ""]],
+        "response_type": ResultType.INDEXED,
+        "response": [[0, "protocol_id", "bytes.decode", ""]],
```

Comment:
why the lower case, space removed? i dont think this is the place to do it

Reply:
Both human format and data format (ie formated with spaces and all lower without spaces) are needed. Didn't really want to have separate fields for these, so had options for converting or not